custom_components/sermatec_inverter/sensor.py

```diff
@@ -390,21 +390,21 @@ async def async_setup_entry(
             device_class    = "power",
             unit            = "W"
         ),
-        SermatecPositivePowerSensor(
+        SermatecBatteryDisChargingPowerSensor(
```

Comment:
A little detail: a lowercase "C" in "Discharging" looks better.

Reply:
yep, copy paste issue ... ^_^